dev/src/webpack/getDefaultWebpackConfig.ts

```diff
@@ -254,17 +262,6 @@ export function getDefaultWebpackConfig(options: DefaultWebpackOptions): webpack
         ]
     };
 
-    // Clean the dist folder before webpacking
-    if (!options.suppressCleanDistFolder) {
```

Comment:
There's an option to suppress it, should we just use that?

Reply:
Uggggh I guess we could. It was nice to delete something that hurt me though.